### ISSUE TITLE
Unable to import picture

### ISSUE THREAD

**Issue by newbie-comfyui** [heshengtao/comfyui_LLM_party#88]
![image](https://github.com/user-attachments/assets/dfa63ff4-18a4-43b9-8bbc-43ed4856a44e)

WARNING: LLM_local_loader.IS_CHANGED() got an unexpected keyword argument 'model_name'
Unused kwargs: ['_load_in_4bit', '_load_in_8bit', 'quant_method']. These kwargs are not used in <class 'transformers.utils.quantization_config.BitsAndBytesConfig'>.
Loading checkpoint shards: 100%|█████████████████████████████████████████████████████████| 2/2 [00:03<00:00,  1.83s/it]
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:128009 for open-end generation.
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Please provide the image you would like me to describe. If you don't have an image in mind, I can suggest a scene for you to describe:

A serene forest path during the golden hour, with sunlight filtering through the dense canopy of trees, casting dappled shadows on the forest floor. A gentle stream runs alongside the path, its crystal-clear waters reflecting the warm hues of the setting sun. The air is filled with the sweet scent of blooming wildflowers, and the distant chirping of birds adds to the tranquil atmosphere.

**Comment by newbie-comfyui** [comment]
HELP ME,PLESE!

**Comment by heshengtao** [comment]
Omost doesn't have vision.

**Comment by heshengtao** [comment]
If you want to use the visual model, you need to use the API version of the node, and then use llama.cpp to transfer or ollama to transfer. Or directly use the visual model such as gpt4o.
